pallets/pools/src/lib.rs

```diff
@@ -46,12 +47,21 @@ pub trait TrancheToken<T: Config> {
 	fn tranche_token(pool: T::PoolId, tranche: T::TrancheId) -> T::CurrencyId;
 }
 
+#[derive(Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug, Default, TypeInfo)]
+pub struct TrancheInput<Rate> {
+	pub interest_per_sec: Option<Rate>,
+	pub min_risk_buffer: Option<Perquintill>,
+	pub seniority: Option<u32>,
```

Comment:
This looks like it's only used for scoring, not for any iteration ordering. In that case, does it make sense to just make this a `weight` field and avoid calculating weights in submission scoring?